images/Dockerfile.broker

```diff
@@ -1,10 +1,17 @@
-from golang:1.10-alpine3.8 as builder
+from golang:1.11-alpine3.8 as builder
 workdir /go/src/github.com/cloudfoundry-incubator/blockhead
 copy . .
 run go build -v -o /broker ./cmd/broker
 
-from alpine:3.8
+
+from node
 expose 3333
+RUN npm install --production --quiet cli-flags
+RUN npm install --production --quiet web3
+RUN npm install --production --quiet solc
```

Comment:
using `solc.js` for the time being is alright. We still need to verify that it works with contracts that require input args though. Maybe not a big deal but just for us to keep in mind.

Reply:
turns out my problem was with the node version. My machine was running `4.6.2` which is really old.

That said, can we lock down the version of our binaries.

node: `0.10.10` seems to work for me
geth: I tested with `1.8.2` and it works but newer versions could be available.

what other binaries do we have to lock down?

Reply:
We can get this in and then you can rebase bind on it to have it to play with, or I can take it out and you can determine what's necessary for bind and make this process available to build on.